crypto/dilithium/pqcrystals-dilithium_dilithium3_ref/symmetric-aes.c

```diff
@@ -0,0 +1,16 @@
+#include <stdint.h>
+#include "symmetric.h"
+#include "aes256ctr.h"
+
+
+#ifdef DILITHIUM_USE_AES
```

Comment:
Would it make more sense to just not include this file? It seems to be if/def'd out completely and this isn't in the upstream code.

Reply:
Removed it in 65221e3. I had included the AES modes so that the flag `DILITHIUM_USE_AES` could be turned on if desired, as I wanted to test the AES KATs also pass. But I can remove this for now and we can decide if the AES modes are something we want to offer later.